packages/redux-router5/index.d.ts

```diff
@@ -0,0 +1,120 @@
+declare module "redux-router5" {
+    import { Router, Plugin, State, Route } from 'router5';
+    import { Store, Middleware, Reducer, Dispatch, Action } from 'redux';
+
+    export function router5Middleware(router: Router): Middleware;
+
+    export interface RouterState {
+        route: State | null;
+        previousRoute: State | null;
+        transitionRoute: State | null;
+        transitionError: any | null;
+    }
+
+    export const router5Reducer: Reducer<RouterState>;
+
+    export function reduxPlugin<TState extends RouterState>(dispatch: Dispatch<TState>): Plugin;
+
+    export function routeNodeSelector<TState extends RouterState>(routeNode: string, reducerKey?: string): (state: TState) => Route;
+
+    // #region actions
+
+    const NAVIGATE_TO = '@@router5/NAVIGATE';
+    const CANCEL_TRANSITION = '@@router5/CANCEL';
+    const TRANSITION_ERROR = '@@router5/TRANSITION_ERROR';
+    const TRANSITION_SUCCESS = '@@router5/TRANSITION_SUCCESS';
+    const TRANSITION_START = '@@router5/TRANSITION_START';
+    const CLEAR_ERRORS = '@@router5/CLEAR_ERRORS';
+    const CAN_DEACTIVATE = '@@router5/CAN_DEACTIVATE';
+    const CAN_ACTIVATE = '@@router5/CAN_ACTIVATE';
+
+    export const actionTypes: {
```

Comment:
Here, the correct pattern seems to be `NAVIGATE_TO: typeof NAVIGATE_TO` or else `actionTypes` keys seen as `any`.

Reply:
Ah correct, thank you!